gobblin-utility/src/main/java/gobblin/util/filesystem/FileSystemInstrumentation.java

```diff
@@ -25,9 +27,36 @@
  */
 public class FileSystemInstrumentation extends FileSystemDecorator {
 
+  protected boolean closed = false;
```

Comment:
Does this need to be thread safe or each object instance will have a different decorator instance?